src/tools/dev/scripts/bv_support/bv_tcmalloc.sh

```diff
@@ -62,10 +62,10 @@ function bv_tcmalloc_ensure
             warn "Unable to build google perftools.  ${TCMALLOC_FILE} not found."
             ANY_ERRORS="yes"
             DO_TCMALLOC="no"
-            if [[ "$DO_SVN" != "yes" ]] ; then
```

Comment:
I don't know what it means to download this through git (or svn as it was before). It seems like this could be removed altogether. I see that it isn't in our third-party repo. I think we should add it to the third-party directory. You can submit this as another ticket if too much effort.

Reply:
FYI...I think google-perftools was somewhat *special* in that its download was *restricted* to only those who had non-anonymous SVN repo read access. I don't know why that was the case but my belief is based on some of the comments I read here and elsewhere. So, I dunno if right thing is to add it to our TPL repo as that would certainly be open to anonymous download.